pliib.hpp

```diff
@@ -350,9 +351,9 @@ namespace pliib{
     }
 
     // TODO convert to template
-    inline std::string join(uint64_t* x, int xlen, char glue){
+    inline std::string join(uint64_t* x, std::size_t xlen, char glue){
```

Comment:
Somehow this change (pliib.hpp:364) has been reverted later. In the last commit the type of `xlen` is still `int` which should be `std::size_t`.

P.S. Sorry to ping you again about this issue. I am developing a tool which uses GFAKluge and the compilation fails because of it.

Reply:
Oops, sorry about this! Just out of curiosity, which compiler are you using that causes failed compilations?

Reply:
I am using gcc 9.2 with `-Werror` flag for some reasons. The flag is responsible for failed compilation which I couldn't limit it to my code in order to ignore warnings in dependencies.